fastqpreprocessing/src/gzstream.cpp

```diff
@@ -0,0 +1,165 @@
+// ============================================================================
```

Comment:
Is this code that you wrote or is this from elsewhere? We should not check code into our repo that is available elsewhere

Reply:
done.